rnd/infra/terraform/modules/static_ip/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "google_compute_address" "static_ip" {
```

Comment:
Where are we using these static ips?

Reply:
these are needed for load balancers. The load balancers are created as part of the ingress on k8s - we specify gce